casepro/cases/views.py

```diff
@@ -101,10 +101,16 @@ def post(self, request, *args, **kwargs):
                 assignee = request.user.get_partner(self.request.org)
                 user_assignee = request.user
 
```

Comment:
Resolved in https://github.com/praekelt/casepro/pull/109/commits/450bbda7090ae545ce5f04337795210ab42f1de8